### PR TITLE
[AAH-3032][AAH-2948] Tasks page + tests

### PR DESCRIPTION
**Pull request by skochay330**
Jira issue: https://issues.redhat.com/browse/AAH-3032, https://issues.redhat.com/browse/AAH-2948

Before: 
Screenshot 2023-12-19 at 12 38 52 PM

After: 
Screenshot 2024-01-04 at 5 48 58 PM

### CODE REVIEW

**Review by nixocio — CHANGES_REQUESTED** [review]
@skochay330 overall good job. 

I noticed that when combining filters for tasks states a 400 is raised.


image

File: frontend/hub/tasks/hooks/useTasksToolbarActions.tsx [hunk 1]
```diff
@@ -39,14 +39,20 @@ export function useStopTasks(onComplete?: (tasks: Task[]) => void) {
   const actionColumns = useMemo(() => [confirmationColumns[0]], [confirmationColumns]);
   const bulkAction = useBulkConfirmation<Task>();
   return useCallback(
-    (tasks: Task[]) => {
+    (ees: Task[]) => {
```
[the remark this answers is not in the document]
Comment (skochay330) [reply frontend/hub/tasks/hooks/useTasksToolbarActions.tsx]: Good catch, thanks!

**Comment by skochay330** [comment]
@nixocio errors when applying multiple filters in the same category is a separate issue, not related to this PR. I have opened a Jira ticket for it [here](https://issues.redhat.com/browse/AAH-3038). For now, I have disabled filtering by name and changed filter by status to single select in order to avoid breakages.

File: .eslintrc.json [hunk 2]
```diff
@@ -50,7 +50,8 @@
           ]
         }
       }
-    ]
+    ],
+    "no-restricted-exports": ["error", { "restrictDefaultExports": { "direct": true } }]
```
Comment (nixocio) [inline .eslintrc.json:54]: @skochay330 this change is already present on the main branch. There are others changes added here that already present on the main branch. Do you intentionally add those again?

Comment (himdel) [reply .eslintrc.json]: rebasing after adding a merge commit is almost impossible :)

I'd suggest starting a new branch from main, and just adding all the intentionally changed files

**Comment by skochay330** [comment]
Closing in favor of https://github.com/ansible/ansible-ui/pull/1468

**Pull request closed without merging**